library/src/main/java/com/getstream/sdk/chat/utils/StorageHelper.kt

```diff
@@ -30,10 +31,8 @@ internal class StorageHelper {
         val cachedFile = File(
             context.cacheDir,
             "STREAM_${
-            dateFormat.format(
-                Date().time
-            )
-            }_${attachmentMetaData.title}"
+            dateFormat.format(Date().time)
```

Comment:
Nit: I'd like to move this hardcoded string to standalone method `fun getFilePath(title: String): String`